auth-api/src/auth_api/resources/task.py

```diff
@@ -44,15 +44,18 @@ def get():
         """Fetch tasks."""
         try:
             # Search based on request arguments
-            task_type = request.args.get('type', None)
-            task_relationship_status = request.args.get('relationshipStatus', None)
-            task_status = request.args.getlist('status', None)
-            page = request.args.get('page', 1)
-            limit = request.args.get('limit', 10)
-
-            response, status = TaskService.fetch_tasks(task_relationship_status=task_relationship_status,
-                                                       task_type=task_type, task_status=task_status,
-                                                       page=page, limit=limit), http_status.HTTP_200_OK
+            task_search = TaskSearch(
```

Comment:
Might be good to use the named parameters like above for this.. I probably should have done that for ORG.

Reply:
done.